js/ad_manager_vast.js

```diff
@@ -740,6 +825,7 @@ OO.Ads.manager(function(_, $) {
      * @private
      * @method Vast#parseLinearAd
      * @param {xml} Xml containing the ad data to be parsed.
+     * @param {number} version The Vast version
```

Comment:
Is this optional? I doesn't look like it being passed into the function ebcause it's only got one param